packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -296,7 +297,14 @@ function ListView({
           startActions={
             <>
               {isSearchable && (
-                <Search
+                // <Search
+                //   label={formatMessage(
+                //     { id: 'app.component.search.label', defaultMessage: 'Search for {target}' },
+                //     { target: headerLayoutTitle }
+                //   )}
+                //   trackedEvent="didSearch"
+                // />
```

Comment:
You still need that one ?

Reply:
I don't indeed, I was supposed to wait to merge the component before integrating it but I needed to test it 😅